MonoGame.Framework/Input/Mouse.Windows.cs

```diff
@@ -31,7 +36,7 @@ private static void PlatformSetPosition(int x, int y)
             PrimaryWindow.MouseState.X = x;
             PrimaryWindow.MouseState.Y = y;
             
-            var pt = Window.PointToScreen(new System.Drawing.Point(x, y));
+            var pt = _window.PointToScreen(new System.Drawing.Point(x, y));
```

Comment:
Sure, we can make Mouse into a Singleton and all static members redirect to `Mouse.Instance`.
If the Dev wants he could create a new Mouse() and assign any window to it.